tools/topology/topology2/cavs/cavs-passthrough-hdmi.conf

```diff
@@ -19,13 +19,23 @@
 <pcm_caps.conf>
 <fe_dai.conf>
 <hda.conf>
+<dmic.conf>
+<pdm_config.conf>
 <hw_config.conf>
 <manifest.conf>
 <route.conf>
 <cavs/common_definitions.conf>
 
 Define {
 	HDA_CONFIG  "none"
+	PLATFORM "tgl"
```

Comment:
I don't understand how this works for CNL, is this redefinting what is in CMakeList.txt

Reply:
the default PLATFORM varaible definition is tgl, and is overridden by CMakelists.txt